src/Symfony/Component/HttpClient/HttpOptions.php

```diff
@@ -63,6 +63,17 @@ public function setQuery(array $query): static
         return $this;
     }
 
+    /**
+     * @return $this
+     */
```

Comment:
You can remove this PHP doc as we have native return type here

Reply:
Rollback, sorry. In fact `$this` is more precise than `static`

Reply:
Reverted ;)